server/hangouts/publications.js

```diff
@@ -82,3 +82,22 @@ Meteor.publish( 'hangoutSearch', function(searchTerm) {
 
     return Hangouts.search(searchTerm);
 });
+
+Meteor.publish('hangoutParticipants', function(room) {
+  check(room, String);
+  if (room) {
+      return AppStats.find({ hangout_id: room });
+  }
+  return this.ready();
+});
+
+Meteor.publish('allHangoutParticipants', function(hangoutRoomIds) {
+  if (hangoutRoomIds == null || typeof hangoutRoomIds === 'undefined' ||
```

Comment:
Please make use of `check` here.
i.e. check(hangoutRoomIds, Match.Maybe([String]))

Reply:
Fixed